test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -37,22 +38,17 @@ Describe "Test-ModuleManifest tests" -tags "CI" {
             "ModuleList"="Modules_InvalidModuleListinModuleManifest";
             "TypesToProcess"="Modules_InvalidManifest";
             "FormatsToProcess"="Modules_InvalidManifest";
-            "RootModule"="Modules_InvalidManifest";
+            #"RootModule"="Modules_InvalidManifest";
             "ScriptsToProcess"="Modules_InvalidManifest"
         }
-        foreach ($parameter in $parametersAndExecptions.Keys) {
+        foreach ($parameter in $parametersAndErrors.Keys) {
+            Write-Warning "Testing $parameter"
```

Comment:
Not sure what the best practice is here.  When doing the test in a loop and something fails, the current error from Pester isn't sufficient to tell you where it failed.  @JamesWTruher suggestions?